src/components/Sankey/utils.ts

```diff
@@ -56,7 +56,7 @@ export function mergeDuplicateChildren(node: any): any {
 	// Merge duplicates and create new children array
 	const mergedChildren: any[] = [];
 
-	Object.entries(displayNameGroups).forEach(([displayName, children]) => {
+	Object.entries(displayNameGroups).forEach(([_displayName, children]) => {
```

Comment:
`59:46  Error: 'displayName' is defined but never used.  @typescript-eslint/no-unused-vars`